### DISCUSSION TITLE
Improve Override Freeze Qty

### DISCUSSION THREAD

**Discussion by harishkarthik7777** [amit0rana/betterOptionsTrading#27] [Ideas]
As of now ,Override freeze qty needs to be enabled each time for buy and sell orders.

1. once it is enabled , it should not get automatically disabled until we disable it manually. 
2. It should not close the sticky order window of Zerodha. (switching from Buy to Sell is easier)

**Comment by amit0rana** [comment]
1. By disable do you mean, one has to click on 'Override Freeze Quantity' checkbox? If yes, that is intentional to avoid mistake. 
2. This one is tricky because I override the action on click of 'buy' or 'sell' button. And I have no way to know whether user has sticky window setting or not. So I can choose to always let it open or always close. Always close is better.

**Reply by harishkarthik7777** [reply]
Both 1,2 would be very much useful for scalpers like me.
Can you give an option in the settings menu..so whoever needs this, can enable and use it.

**Reply by harishkarthik7777** [reply]
or please help me if  there is a way to do this only for myself.
